CHANGELOG.rst

```diff
@@ -20,19 +20,21 @@ This document describes changes between each past release.
 **Breaking changes**
 
 - Renamed internal backend classes for better consistency. Settings remain
-  unchanged (#).
+  unchanged (#491).
```

Comment:
Add a detail like about what happened in Syncto: *If you imported the backend classes in your Cliquet application, it will break*